data/events/2024/belo-horizonte/main.yml

```
@@ -160,14 +163,14 @@ team_members: # Name is the only required field for team members.
 
  
 
-organizer_email: "belo-horizonte@devopsdays.org" # Put your organizer email address here
+organizer_email: "devopsdaysbhz@gmail.com" # Put your organizer email address here
```

This email address should stay as the group email provided by @devopsdays.org:
```suggestion
organizer_email: "belo-horizonte@devopsdays.org" # Put your organizer email address here
```